.github/workflows/move-to-done.yaml

```diff
@@ -0,0 +1,80 @@
+name: move-to-done
```

Comment:
We initially thought about combining this with the `sync-board` workflow, since that is nicely generically named, but it would have basically been two completely different workflows in one file. 

If you're interested in us renaming the `sync-board` workflow to be _less_ generic, since it can't be used as universally as we initially thought, let us know!

Reply:
I think renaming makes a ton of sense here. Do you have any suggestions? `sync-review`? Maybe @christinaausley has a good idea.